### PR TITLE
Update syntax highlight styles

### PR DESCRIPTION
**Pull request by lornajane**
Looking at #761 and digging around in the codebase, I found `pygments2style.py` which does indeed contain the code that takes in whatever pygments styles are on your computer and turn them into rst2pdf styles. So I've started by dumping the output of that into this branch, it needs more testing and also some investigation into what's now available in pygments that we might not already have support for in rst2pdf.

### CODE REVIEW

**Comment by lornajane** [comment]
Updated to say that the newly generated/updated styles do seem to work pretty well. Going to put this PR to not-draft status in case anyone else has an opinion on what else we should do before merging. I'll add a list of styles to the documentation I think as I'm not sure how else they can be discovered.

File: doc/manual.rst [hunk 1]
```diff
@@ -955,20 +955,43 @@ gray. For example, to highlight ``print "line a"`` and ``print "line b"``:
...
 
+* ``abap``
+* ``algol_nu``
+* ``algol``
+* ``arduino``
 * ``autumn``
 * ``borland``
 * ``bw``
 * ``colorful``
+* ``default``
```
Comment (akrabat) [inline doc/manual.rst:966]: Is this style picked up as the default colour scheme if the user doesn't explicitly choose another one?

**Comment by lornajane** [comment]
Rebased to eliminate conflicts from style fixes

File: doc/manual.rst (hunk 1 above)
Comment (lornajane) [reply doc/manual.rst:966]: Yes, this seems to be our default as well as the pygments default style (which is updated by this PR). I get the same output with and without the "default" stylesheet.

**Comment by lornajane** [comment]
Do we need to do anything else before we can merge this styles PR?

**Pull request closed without merging**